app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: "Teach pupils with special educational needs and disabilities"
-heading: "Teach pupils with special educational needs and disabilities (SEND)"
+heading: "Teach pupils with special educational needs and disabilities"
 description: |-
   Find out how you can teach pupils with special educational needs and disabilities and how to become a special educational needs coordinator (SENCO).
```

Comment:
Is there any merit in including `(SEND)` in the meta description now that it's been removed from the h1?